src/java.base/share/classes/java/lang/Class.java

```diff
@@ -3021,6 +3021,37 @@ private void checkPackageAccess(SecurityManager sm, final ClassLoader ccl,
         }
     }
 
+    /*
+     * Checks if a client loaded in ClassLoader ccl is allowed to access the provided
+     * classes under the current package access policy. If access is denied,
+     * throw a SecurityException.
+     *
+     * NOTE: this method should only be called if a SecurityManager is active
+     *       classes must be non-empty
+     *       all classes provided must be loaded by the same ClassLoader
+     * NOTE: this method does not support Proxy classes
+     */
+    private static void checkPackageAccessForPermittedSubclasses(SecurityManager sm,
```

Comment:
@stuart-marks You should be able to resolve this comment. This pr is one in lahodaj/jdk, and you have GitHub write access to this repo.

Reply:
Resolving.